fortinet_fortigate/plugin.spec.yaml

```diff
@@ -674,36 +674,36 @@ actions:
         description: Optional name to filter on
         type: string
         required: false
+        example: 198.51.100.100
     output:
       address_objects:
         title: Address Objects
         description: A list of address objects
         type: '[]object'
         required: true
-
   create_address_object:
     title: Create Address Object
     description: Create an address object
     input:
-      host:
-        title: Host
+      address:
+        title: Address
         description: The host to create. This can be an IP address, CIDR IP address e.g. 198.51.100.0/24, or a domain name
```

Comment:
```suggestion
        description: The address to assign to the Address Object. This can be an IP address, CIDR IP address e.g. 198.51.100.0/24, or a domain name
```